Algorithms/Numeric/Decomposition/SVD.cs

```diff
@@ -0,0 +1,332 @@
+using System;
+
+namespace Algorithms.Numeric.Decomposition
+{
+    /// <summary>
+    /// Singular Vector Decomposition decomposes any general matrix into its
+    /// singular values and a set of orthonormal bases.
+    /// </summary>
+    public static class SVD
```

Comment:
Codacy found an issue: [Rename class 'SVD' to match pascal case naming rules, consider using 'Svd'.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6318623)